addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -459,6 +456,26 @@ const Link = Widget.extend({
      * @param {boolean} [active]
      */
     _setSelectOption: function ($option, active) {},
+    /**
+     * Set the link from data.
```

Comment:
"data" is not very descriptive

Reply:
Also "the link", that would usually be interpreted as the URL but here we're setting the label / content of the link element.

Reply:
updated